src/transceivercontroller/TransceiverController.ts

```diff
@@ -78,7 +78,13 @@ export default interface TransceiverController {
   localVideoTransceiver(): RTCRtpTransceiver;
 
   /**
-   * Set [[RTCRtpEncodingParameters]] on the sender of transceiver
+   * Set [[RTCRtpEncodingParameters]] on the sender of transceiver.
+   * This method is called when a video is enabled/disabled or when there is the active speaker changes.
+   * The input parameters should not be empty.
```

Comment:
"when the active speaker changes".

But here you're describing _what the default policy does_, right?